dist/ios/CdrSize.m

```diff
@@ -9,77 +9,99 @@
-float const CdrProminenceFloatingSpread = 0.00f;
-float const CdrProminenceLiftedX = 256.00f;
-float const CdrProminenceLiftedY = 256.00f;
-float const CdrProminenceLiftedBlur = 256.00f;
-float const CdrProminenceLiftedSpread = 0.00f;
-float const CdrRadiusSharp = 0.00f;
-float const CdrRadiusSoft = 32.00f;
-float const CdrRadiusSofter = 64.00f;
-float const CdrSpaceEighthX = 32.00f;
-float const CdrSpaceQuarterX = 64.00f;
-float const CdrSpaceHalfX = 128.00f;
-float const CdrSpaceThreeQuarterX = 192.00f;
-float const CdrSpaceOneX = 256.00f;
-float const CdrSpaceOneAndAHalfX = 384.00f;
-float const CdrSpaceTwoX = 512.00f;
-float const CdrSpaceFourX = 1024.00f;
-float const CdrSpaceInsetEighthX = 32.00f;
-float const CdrSpaceInsetEighthXSquishTopBottom = 0.00f;
-float const CdrSpaceInsetEighthXSquishLeftRight = 32.00f;
-float const CdrSpaceInsetEighthXStretchTopBottom = 64.00f;
-float const CdrSpaceInsetEighthXStretchLeftRight = 32.00f;
-float const CdrSpaceInsetQuarterX = 64.00f;
-float const CdrSpaceInsetQuarterXSquishTopBottom = 32.00f;
-float const CdrSpaceInsetQuarterXSquishLeftRight = 64.00f;
-float const CdrSpaceInsetQuarterXStretchTopBottom = 96.00f;
-float const CdrSpaceInsetQuarterXStretchLeftRight = 64.00f;
-float const CdrSpaceInsetHalfX = 128.00f;
-float const CdrSpaceInsetHalfXSquishTopBottom = 64.00f;
-float const CdrSpaceInsetHalfXSquishLeftRight = 128.00f;
-float const CdrSpaceInsetHalfXStretchTopBottom = 192.00f;
-float const CdrSpaceInsetHalfXStretchLeftRight = 128.00f;
-float const CdrSpaceInsetThreeQuarterX = 192.00f;
-float const CdrSpaceInsetThreeQuarterXSquishTopBottom = 96.00f;
-float const CdrSpaceInsetThreeQuarterXSquishLeftRight = 192.00f;
-float const CdrSpaceInsetThreeQuarterXStretchTopBottom = 288.00f;
-float const CdrSpaceInsetThreeQuarterXStretchLeftRight = 192.00f;
-float const CdrSpaceInsetOneX = 256.00f;
-float const CdrSpaceInsetOneXSquishTopBottom = 128.00f;
-float const CdrSpaceInsetOneXSquishLeftRight = 256.00f;
-float const CdrSpaceInsetOneXStretchTopBottom = 384.00f;
-float const CdrSpaceInsetOneXStretchLeftRight = 256.00f;
-float const CdrSpaceInsetOneAndAHalfX = 384.00f;
-float const CdrSpaceInsetOneAndAHalfXSquishTopBottom = 192.00f;
-float const CdrSpaceInsetOneAndAHalfXSquishLeftRight = 384.00f;
-float const CdrSpaceInsetOneAndAHalfXStretchTopBottom = 576.00f;
-float const CdrSpaceInsetOneAndAHalfXStretchLeftRight = 384.00f;
-float const CdrSpaceInsetTwoX = 512.00f;
-float const CdrSpaceInsetTwoXSquishTopBottom = 256.00f;
-float const CdrSpaceInsetTwoXSquishLeftRight = 512.00f;
-float const CdrSpaceInsetTwoXStretchTopBottom = 768.00f;
-float const CdrSpaceInsetTwoXStretchLeftRight = 512.00f;
-float const CdrSpaceInsetFourX = 1024.00f;
-float const CdrSpaceInsetFourXSquishTopBottom = 512.00f;
-float const CdrSpaceInsetFourXSquishLeftRight = 1024.00f;
-float const CdrSpaceInsetFourXStretchTopBottom = 1536.00f;
-float const CdrSpaceInsetFourXStretchLeftRight = 1024.00f;
+float const CdrIconSizeSm = 16;
```

Comment:
Should these have units? If so, then it looks like a transform isn't working as intended

Reply:
Ideally yes, but I am still creating a transform to convert them.  According to the iOS docs, they will get converted automatically by xcode.

Reply:
For some reason, the remToPx trasform would correctly convert to float, but it scales by 16

Reply:
You need to make a custom one then since ours is based off of 10. I had to do that for `transforms/size/px-to-rem.js` too.

Reply:
Created the custom transform and it appears to work properly.